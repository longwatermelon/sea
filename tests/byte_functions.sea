fn add_bytes(a:byte, b:byte) -> byte {
    return a + b;
}

fn main() -> int {
    let x:byte = 10b;
    let y:byte = 25b;
    let result:byte = add_bytes(x, y);
    return result;
}
