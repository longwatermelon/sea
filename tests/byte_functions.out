RETURN_CODE:35
STDOUT:
